README.md

```diff
@@ -33,19 +33,19 @@ A sample app for testing purposes is automatically built after commits in the re
 ### Screenshots of a sample Android app
 
 <p align="center">
-<img src="docs/images/app-drawer.png" height = "480" width="270"> <img src="docs/images/app-tracks.png" height = "480" width="270"> <img src="docs/images/app-schedule.png" height = "480" width="270">
+<img src="docs/_static/images/app-drawer.png" height = "480" width="270"> <img src="docs/_static/images/app-tracks.png" height = "480" width="270"> <img src="docs/_static/images/app-schedule.png" height = "480" width="270">
```

Comment:
These images aren't rendering horizontally in the generated docs. Use `table` tag.